demo/campaign/src/components/IndentedSectionGroup.tsx

```diff
@@ -7,18 +7,31 @@ export interface IndentedSectionGroupProps extends React.ComponentProps<typeof M
     cssClass?: string;
 }
 
-const CONTENT_WIDTH = theme.mailSize.contentWidth - theme.mailSize.contentSpacing * 2;
+export const indentedSectionGroupStyles = (
+    <MjmlStyle>{`
```

Comment:
Please use the css helper for better highlighting and prettier formatting

```suggestion
    <MjmlStyle>{css`
```